### DISCUSSION TITLE
How do you run this?

### DISCUSSION THREAD

**Discussion by Plawro** [helpLost/leto#1] [Questions]
Hello, I tried to run your game by opening the exe, got an error, tried to open the main.cpp in VS, but I can't manage to run it.

**Comment by helpLost** [comment]
Hello, thanks for trying Leto! Unfortunately this is still WAYY in Alpha, so nothing can be guaranteed to work outside of my machine. Even here, there isn't much to look at. If you're really dedicated to trying to run it, I can try to help (via screenshots) but it would honestly be more worth your time to star the repo and come back in a couple months. So sorry for the troubles.

**Reply by Plawro** [reply]
Oh, alright, but inform me about the updates!! Good luck with the project! (also there is nothing such as Intel i8 processor :D)

**Reply by helpLost** [reply]
Oh my, that's 100% my mistake. I mistyped i9--and it really should be i7-12700 instead!
